mlcomp/parallelm/mlpiper/main.py

```diff
@@ -109,13 +109,17 @@ def _add_deploy_sub_parser(subparsers, sub_parser_name, sub_parser_help):
     parser_prepare.add_argument('--force', action='store_true',
                                 help='Overwrite any previous generated files/directories (.e.g deployed dir)')
 
+    parser_prepare.add_argument('--mlcomp-jar', default=None, help="Path to mlcomp jar")
```

Comment:
As we've already discussed we should do it by distributing the mlcomp jar along with the ml-comp wheel package. This option is far for being straightforward for a data-scientist who works with this tools - how to support upgrades and so forth. I would invest some time to accomplish it properly.